README.md

```diff
@@ -17,33 +28,15 @@ To set up the development environment for this website, you'll need to install t
   Run `nvm use` to activate the correct version.
 
 After these basic requirements are met, run the following commands in the website's folder:
-```
-$ npm install
-```
-
-### Getting started
-
-Source code goes in `app` and after building it will be copied to `dist`.
 
 ```
-$ npm run serve
+npm install
 ```
-Compiles the sass files, javascripts, and launches the server making the site available at `http://localhost:3000/`
-The system will watch files and execute tasks whenever one of them changes.
-The site will automatically refresh since it is bundled with livereload.
 
-### Other commands
-```
-$ npm run build
-```
-Builds the app for production, and puts it into `/dist`.
+### Getting started
 
-```
-$ npm run lint
-```
-Lints the app according with the defined style.
+#### Environment Variables
 
-## Configuration
 The configuration options are defined in `app/assets/scripts/config.js`.
```

Comment:
It's not really in `config.js` but in the appropriate file in the `config` folder. Should we mention this or the notes on that file are enough?